scripts/ananse

```diff
@@ -231,6 +239,18 @@ if __name__ == "__main__":
         action=NegateAction,
         nargs=0,
     )
+    
+    group.add_argument(
+        "--necessary-output",
+        "--detailed-output",
+        default=True,
+        help="generate simple or detaild output in the GRN file "
+             "By default only necessary data is stored",
+        dest="necessary_output",
+        action=NegateAction,
```

Comment:
I _think_ you inverted the option here. By writing either `--necessary-output` or `--detailed-output` you set `necessary_output=False` now(?)

Reply:
Its default is True, if you add --necesary-output (or -no) its True, if you add '--detailed-output (or --full-output) it because False.

I'll swap the option that its basically: --full-output (default = False), if you add --full-output  it becomes True, because this indeed makes more sense!

Reply:
ahh, I see you used a `NegateAction` argument. Those are terrible and shouldn't be used IMO.

Take a look at this flag: without the flag you get the default setting, and with the flag you get the alternate. Simple and clean when you have a boolean setting. 

```
    group.add_argument(
        "-p",
        "--plot",
        dest="plot",
        help="Create influence score plot.",
        action="store_true",
        default=False,
    )
```

Reply:
Just finished fixing the param by looking at the plot function, before I saw this helpfull comment about the plot function xD